features/admin.identity-providers.v1/meta/authenticator-meta.ts

```diff
@@ -90,7 +90,10 @@ export class AuthenticatorMeta {
             "of active user sessions.",
             [ LocalAuthenticatorConstants.AUTHENTICATOR_IDS
                 .X509_CERTIFICATE_AUTHENTICATOR_ID ]: "Authenticate clients using " +
-                "client certificates."
+                "client certificates.",
```

Comment:
Let's not add any meta details in this identity-providers feature module.

Reply:
I think this is to show the push authenticator in the add authentication modal in the login flow tab. @ZiyamSanthosh could you please confirm?

Reply:
We still use AuthenticatorFormFactory from identity-providers feature to render the authenticator's edit form. Therefore we still need to add these to identity providers module

[1] https://github.com/wso2/identity-apps/blob/master/features/admin.connections.v1/components/edit/edit-multi-factor-authenticator.tsx#L24